modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -87,11 +91,17 @@ If you disable simultaneous multithreading, ensure that your capacity planning a
 //====
 //<6> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 //endif::openshift-origin[]
-<4> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<5> The name of an existing resource group.
-<6> Required. The installation program prompts you for this value.
+<4> The smtLevel specifies the level of SMT to set to the control plane and compute machines. The supported values are 1, 2, 4, 8, 'off' and 'on'. The default value is 8. The smtLevel 'off' sets SMT to off and smtlevel 'on' sets SMT to the default value 8 on the cluster nodes.
```

Comment:
There is an inconsistency.  The statement in the bare-metal document states that the level can be any value between 1 and 8, and here we are specific to powers of 2.  We should be consistent.

Reply:
@manojnkumar My apologies. Updated the bare metal file with the correct value.